.github/workflows/build_pkg.yml

```diff
@@ -53,7 +53,7 @@ jobs:
         run: conda list
 
       - name: Build from local conda recipe
-        run: conda mambabuild .
+        run: conda mambabuild -c conda-forge .
```

Comment:
I am not sure why the previously available channels are no longer detected here but this seems to fix the build issues

Reply:
Interesting. I thought it was covered by the recipe that contains conda-forge. Good that you found a solution. Didn't know what to try else.